librz/core/cmd/cmd_print.c

```diff
@@ -5266,16 +5046,6 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
 					l, len, 'i');
 			}
 			break;
-		case 'j': // pij is the same as pdj
-			if (l != 0) {
-				cmd_pdj(core, input + 2, block);
-			}
-			break;
-		case 'd': // "pid" is the same as pdi
-			if (l != 0) {
-				rz_core_disasm_pdi(core, l, 0, 0);
-			}
-			break;
```

Comment:
where are these going?

Reply:
https://github.com/rizinorg/rizin/pull/2552
They're newshells now, just some old styles code hasn't been removed

Reply:
I think it's fine then 👍